pootle/apps/pootle_store/unit/proxy.py

```diff
@@ -39,3 +40,18 @@ def hasplural(self):
             self.source is not None
             and (len(self.source.strings) > 1
                  or getattr(self.source, "plural", None)))
+
+    def getnotes(self, origin=None):
+        return self.unit["%s_comment" % origin]
+
+    def getcontext(self):
```

Comment:
I used UnitProxy as base class but I didn't use any of its methods here
https://github.com/translate/pootle/pull/5202/commits/a56c885d290f12cda93cf25ff624bbbcb0775a54